routers/api/v1/api.go

```diff
@@ -642,11 +642,11 @@ func RegisterRoutes(m *macaron.Macaron) {
 					}, reqGitHook(), context.ReferencesGitRepo(true))
 				}, reqToken(), reqAdmin())
 				m.Group("/collaborators", func() {
-					m.Get("", repo.ListCollaborators)
-					m.Combo("/:collaborator").Get(repo.IsCollaborator).
+					m.Get("", reqAnyRepoReader(), repo.ListCollaborators)
+					m.Combo("/:collaborator", reqAdmin()).Get(repo.IsCollaborator).
 						Put(bind(api.AddCollaboratorOption{}), repo.AddCollaborator).
 						Delete(repo.DeleteCollaborator)
```

Comment:
You should also add reqAdmin on `Put` and `Delete`.

Reply:
Yes. I think ListCollaborators and IsCollaborator should be the same required permission.

Reply:
@davidsvantesson Yes I agree. I have made the appropriate changes.